pkg/command/service/clean.go

```diff
@@ -0,0 +1,125 @@
+package service
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/oidc"
+
+	servingv1client "knative.dev/serving/pkg/client/clientset/versioned/typed/serving/v1"
+
+	"github.com/spf13/cobra"
+
+	"github.com/zhanggbj/kperf/pkg"
+	"github.com/zhanggbj/kperf/pkg/generator"
+)
+
+func NewServiceCleanCommand(p *pkg.PerfParams) *cobra.Command {
+	ksvcCleanCommand := &cobra.Command{
+		Use:   "clean",
+		Short: "clean ksvc",
+		Long: `clean ksvc workload
+
+For example:
+# To clean ksvc workload
+kperf service clean --nsprefix testns/ --ns nsname
+`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			var nsRangeMap map[string]bool = map[string]bool{}
+			if nsPrefix != "" {
+				r := strings.Split(nsRange, ",")
+				if len(r) != 2 {
+					fmt.Printf("Expected Range like 1,500, given %s\n", nsRange)
+					os.Exit(1)
+				}
+				start, _ := strconv.Atoi(r[0])
+				end, _ := strconv.Atoi(r[1])
+				if start >= 0 && end >= 0 && start <= end {
+					for i := start; i <= end; i++ {
+						nsRangeMap[fmt.Sprintf("%s%d", nsPrefix, i)] = true
```

Comment:
`nsRangeMap` in clean.go is %s%d, while in generate.go is %s-%d. They should be matched.

Reply:
@gyohuangxin nice catch, thanks! This will be refined in https://github.com/knative-sandbox/kperf/issues/4 after the initial PR merged.

Reply:
Why not fix it right here?